File: src/main/java/net/softbell/bsh/dto/request/MemberGroupPermissionDto.java
package net.softbell.bsh.dto.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : 회원 그룹 권한 정보 DTO
 */
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class MemberGroupPermissionDto
{
    private Long nodeGid;
    private Integer permission;
}

File: src/main/kotlin/net/softbell/bsh/dto/request/MemberGroupPermissionDto.kt
package net.softbell.bsh.dto.request

/**
 * @author : Bell(bell@softbell.net)
 * @description : 회원 그룹 권한 정보 DTO
 */
data class MemberGroupPermissionDto (
        var nodeGid: Long,
        var permission: Int
)